cfme/ansible/playbooks.py

```diff
@@ -4,24 +4,21 @@
 from navmazing import NavigateToAttribute, NavigateToSibling
 from widgetastic.widget import Text, View
 from widgetastic_patternfly import Button, Dropdown
-
-from cfme.base import Server
-from cfme.base.login import BaseLoggedInPage
-from cfme.exceptions import ItemNotFound
-from cfme.modeling.base import BaseCollection, BaseEntity
-from cfme.utils.appliance.implementations.ui import navigator, navigate_to, CFMENavigateStep
 from widgetastic_manageiq import (
     BaseEntitiesView,
-    BaseListEntity,
-    BaseQuadIconEntity,
-    BaseTileIconEntity,
     BreadCrumb,
     ItemsToolBarViewSelector,
-    NonJSBaseEntity,
     PaginationPane,
     SummaryTable,
 )
 
+from cfme.base import Server
```

Comment:
These were in the correct location before, as widgetastic_manageiq is a local import.